### PR TITLE
Add bot-2 icon

### LINKED ISSUE #934: Better robot icon
The bot icon is not very obvious. It could just as easily represent a joystick (from the profile) or perhaps an RC controller (from the top).

More obvious eyes, mouth, ears, neck, or body could help. Random example from a google search:

download

### PR DESCRIPTION
**Pull request by ericfennis**
closes #934 

## What is the purpose of this pull request?
- [x] New Icon
- [ ] Bug fix
- [ ] New Feature
- [ ] Documentation update
- [ ] Other:

### Description
This PR solves the issue of #934 for requesting a better bot icon. The current bot icon is not always best looking for the new area of AI Apps. Since my design got most of the thumbs ups, here the PR.

### Icon use case
Ai generated content, same usecase as current `bot` icon

### Alternative icon designs 
See alternative designs in #934 

### Concept 

- [x] I have provided valid use cases for each icon.
- [x] I have not added any a brand or logo icon.
- [x] I have not used any hate symbols.
- [x] I have not included any religious or political imagery.

### Author, credits & license

- [x] The icons are solely my own creation.
- [x] The icons were originally created in #934 by @ericfennis
- [x] I've based them on the following Lucide icons: bot
- [x] I've based them on the following design: 

### Naming 

- [x] I've read and followed the [naming conventions](https://lucide.dev/guide/design/icon-design-guide#naming-conventions). 
- [x] I've named icons by what they are rather than their use case.
- [x] I've provided meta JSON files in `icons/[iconName].json`.

### Design 

- [x] I've read and followed the [icon design guidelines](https://lucide.dev/guide/design/icon-design-guide)
- [x] I've made sure that the icons look sharp on low DPI displays.
- [x] I've made sure that the icons look consistent with the icon set in size, optical volume and density.
- [x] I've made sure that the icons are visually centered.
- [x] I've correctly optimized all icons to two points of precision.

## Before Submitting 

- [x] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [x] I've checked if there was an existing PR that solves the same issue.

### CODE REVIEW

**Comment by ericfennis** [comment]
This one violates the naming convention of using numbers for alternative icons.
I seriously don't know what alternative name we should use, suggestions are welcome!

**Comment by karsa-mistmere** [comment]
> This one violates the naming convention of using numbers for alternative icons. I seriously don't know what alternative name we should use, suggestions are welcome!

I don't think we should use any, but instead simply replace the existing robot icon.

File: icons/bot-2.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "ericfennis"
```
Comment (jguddas) [inline icons/bot-2.json:4]:
```suggestion
    "mittalyashu",
    "ericfennis"
```

**Review by jguddas — APPROVED** [review]
> I don't think we should use any, but instead simply replace the existing robot icon.

Strongly agree, otherwise :+1:

**Comment by ericfennis** [comment]
@karsa-mistmere @jguddas Replaced it!

**Review by karsa-mistmere — APPROVED** [review]

File: icons/bot.json [hunk 2]
```diff
@@ -1,7 +1,6 @@
 {
   "$schema": "../icon.schema.json",
   "contributors": [
-    "mittalyashu",
```
Comment (jguddas) [inline icons/bot.json:4]: Why remove them?

Comment (karsa-mistmere) [reply icons/bot.json:4]: Because their entire contribution to this icon has essentially been removed?

Comment (jguddas) [reply icons/bot.json:4]: Okay 👍

**Pull request merged**